packages/components/src/Persona.purs

```diff
@@ -375,6 +376,14 @@ type Address =
   , apartment     :: Nullable String
   }
 
+type TemporaryAddressChange =
```

Comment:
The Persona `PastTemporaryAddress` has the type
```
countryCode* | string
zipcode* | string
cityName | string
street* | string
temporaryName | string
```

Should we mimic that here?

Reply:
Meaning, by Personas standards, `countryCode`, `zipCode` and `street` should always exist.

Reply:
I made the change, though I ended up using `Just` for them when I translated them to `AddressChange` type. I think we're better of sticking to `Maybe` for that type's fields since that reflects how input fields store their values in them.